release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1196,11 +1159,6 @@ In the following tables, features are marked with the following statuses:
 |====
 |Feature |4.16 |4.17 |4.18
 
-|Hosted control planes for {product-title} on {aws-first}
-|Technology Preview
-|Technology Preview
-|Technology Preview
-
 |Hosted control planes for {product-title} using non-bare metal agent machines
 |Technology Preview
 |Technology Preview
```

Comment:
I can't seem to comment directly on lines 1207 - 1185, so I'm commenting here.

- On line 1167, `Hosted control planes` can be replaced with `{hcp-capital}`
- HCP on ARM64 went GA in 4.17, so lines 1169 and 1170 should state `General Availability`
- HCP on IBM Power and HCP on IBM Z went GA in 4.17, so lines 1174, 1175, 1179, and 1180 should state `General Availability`

FYI @xenolinux @SNiemann15

Reply:
Thanks! Updated.